src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(':')
+    .flatMap((e) => e.split(';'))
```

Comment:
The use of `flatMap` is unnecessary here. You can use `map` instead since you are not flattening nested arrays but simply splitting each element.